decidim-elections/app/views/layouts/decidim/admin/voting.html.erb

```diff
@@ -93,6 +93,11 @@
         <span class="callout-text"><%= t("votings.edit.add_election_component", scope: "decidim.votings.admin") %></span>
       </div>
     <% end %>
+    <% if current_participatory_space.polling_stations_with_missing_officers? %>
+      <div class="callout alert callout--full">
+        <span class="callout-text"><%= t("votings.edit.assign_missing_officers", scope: "decidim.votings.admin") %></span>
+      </div>
```

Comment:
Maybe here you could use the [`Decidim::AnnouncementCell`](https://github.com/decidim/decidim/blob/develop/decidim-core/app/cells/decidim/announcement_cell.rb) so it's easier to maintain/customize appearance.

```suggestion
      <%= cell("decidim/announcement", t("votings.edit.assign_missing_officers", scope: "decidim.votings.admin") %>
```

Reply:
Oh nice! Didn't know about that!
Thanks @agustibr!